include/zephyr/drivers/wifi/nrfwifi/off_raw_tx/off_raw_tx_api.h

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/** @file
+ *
+ * @addtogroup nrf70_off_raw_tx_api nRF70 Offloaded raw TX API
+ * @{
+ *
+ * @brief File containing API's for the Offloaded raw TX feature.
+ */
+
+#ifndef INCLUDE_ZEPHYR_DRIVERS_OFF_RAW_TX_API_H_
+#define INCLUDE_ZEPHYR_DRIVERS_OFF_RAW_TX_API_H_
+
+#include <stdbool.h>
+#include <stdint.h>
```

Comment:
these would cause issues if we build for Linux

Reply:
But this is a zephyr specific file?